docs/updating.rst

```diff
@@ -2,35 +2,55 @@
 How to update the Atomic Developer Bundle to the latest version
 ===============================================================
 
-* If you are using the image from atlas.hashicorp.com
+Depending on your implementation of the ADB, you might be able to directly
+update your Vagrant box. In some cases, you will need to perform a clean
+installation or follow additional steps to complete the update.
 
-  1. Check if a newer version of the box is available use this command::
+--------------------------------------------------
+Update the ADB for images from atlas.hashicorp.com
```

Comment:
@bexelbie I try to avoid gerunds in task titles, but maybe that's a personal/historical style preference.

Maybe "Update ADB from atlas.h..."?